e2e_testing/torchscript/conv.py

```diff
@@ -133,3 +133,59 @@ def forward(self, x):
 def Conv2dWithPaddingDilationStrideStaticModule_basic(module, tu: TestUtils):
     t = tu.rand(5, 2, 10, 20)
     module.forward(t)
+
+# ==============================================================================
+
+# class ConvolutionModule1D(torch.nn.Module):
+#     def __init__(self):
+#         super().__init__()
+
+#     @export
+#     @annotate_args([
+#         None,
+#         ([-1, -1, -1], torch.float32, True),
+#         ([-1, -1, -1], torch.float32, True),
+#     ])
+#     def forward(self, inputVec, weight):
+#         return torch.ops.aten.convolution(inputVec, weight, bias=None, stride=[1], padding=[0], dilation=[1], transposed=False, output_padding=[0], groups=1)
+
+# @register_test_case(module_factory=lambda: ConvolutionModule1D())
+# def ConvolutionModule1D_basic(module, tu: TestUtils):
+#     with torch.backends.mkldnn.flags(enabled=False):
+#         module.forward(torch.ones(3, 3, 10), torch.ones(3, 3, 2))
+
+class ConvolutionModule2D(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1, -1], torch.float32, True),
+        ([-1, -1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, inputVec, weight):
+        return torch.ops.aten.convolution(inputVec, weight, bias=None, stride=[1, 1], padding=[0, 0], dilation=[1, 1], transposed=False, output_padding=[0, 0], groups=1)
+
+@register_test_case(module_factory=lambda: ConvolutionModule2D())
+def ConvolutionModule2D_basic(module, tu: TestUtils):
+    with torch.backends.mkldnn.flags(enabled=False):
```

Comment:
why is this line needed?

Reply:
We've had some issues with running certain e2e tests because the mkldnn backend doesn't support some ops. I think convolution was one of those, but it may have been convolution_backward.

Reply:
I missed this the last time. This code shouldn't be part of our e2e test right? because we don't use mkldnn backend in our e2e testing.